File: test_archives/py_functions/batch2/function3.py
def function3(string1, int1, float1):
    """This function takes in 3 parameters and returns a value. 
    Parameters:
    string1 (str): A string value
    int1 (int): An integer value
    float1 (float): A float value
    Returns:
    int2 (int): An integer value
    """
    int2 = int1 * float1
    if string1 == "yes":
        int2 += 10
    elif string1 == "no":
        int2 -= 5
    return int2

File: test_archives/cpp_functions/batch2/function3.cpp
# C++ Code

#include <iostream>
#include <string>

int function3(std::string string1, int int1, float float1)
{
    int int2 = int1 * float1;
    if (string1 == "yes")
    {
        int2 += 10;
    }
    else if (string1 == "no")
    {
        int2 -= 5;
    }
    return int2;
}